src/components/plotters/plot2D.js

```diff
@@ -0,0 +1,174 @@
+import React, { useEffect, useState } from "react";
+import * as THREE from "three";
+import { eventBus } from "../../event-bus/eventBus";
+
+const WIDTH = window.innerWidth,
+  HEIGHT = window.innerHeight,
+  VIEW_ANGLE = 50,
+  ASPECT = WIDTH / HEIGHT,
+  NEAR = 0.1,
+  FAR = 10000,
+  SEGMENTS = 80;
+
+const FunctionPlotter2D = ({
```

Comment:
Would separate plotter logic from react scope. 
You can create a: Logger logic where at each population tick the executor will dump the population to the Logger and depending on the executing context (react/node) you will get information shown on the screen.
- node logger - print out the best execution function
- table logger - return HTML DOM section with table (best *x* configurable individuals)
- function plotter - return THREE context DOM Element

in order to integrate them into react you can create a DOM element for Graphical Loggers using `window.createElement` - this will allow you to reuse the same DOM instance instead of allowing it to be controlled by React. The React Library can be used to render the wrapping application (function selection, execution period (iterations), hyperparameters, etc.)